testcafe/tests/pages/photos-model.js

```diff
@@ -48,21 +36,48 @@ export default class Page {
     this.modalDelete = Selector('[class*="c-modal"]').find('div')
     this.modalDeleteBtnDelete = this.modalDelete.find('button').nth(2) //REMOVE
 
-    this.allPhotos = Selector('[class*="pho-photo"]').find(
-      '[class*="pho-photo-item"]'
+    this.allPhotosWrapper = this.photoSection.find('[class^="pho-photo"]')
+    this.allPhotos = Selector('img[class*="pho-photo-item"]')
+
+    // Photo fullscreen
+    this.photoFull = Selector('[class*="pho-viewer-imageviewer"]').find('img')
+    this.photoNavNext = Selector('[class*="pho-viewer-nav--next"]')
+    this.photoNavNextBtn = this.photoNavNext.find(
+      '[class*="pho-viewer-nav-arrow"]'
+    )
+    this.photoNavPrevious = Selector('[class*="pho-viewer-nav--previous"]')
+    this.photoNavPreviousBtn = this.photoNavPrevious.find(
+      '[class*="pho-viewer-nav-arrow"]'
     )
+    this.photoBtnClose = Selector('[class*="pho-viewer-toolbar-close"]').find(
+      '[class*="c-btn"]'
```

Comment:
Pour être franc, je trouve les sélecteurs bien plus parlants que les noms de variable que l'on créés. 

On pourrait pas, par la suite, plutôt que de faire des : 
```js
this.photoBtnClose = Selector('[class*="pho-viewer-toolbar-close"]').find(
      '[class*="c-btn"]'
```

faire 
``` 
const PHO-VIEWER-TOOLBAR-CLOSE = 'pho-viewer-toolbar-close'
//et dans le code plus loin utiliser non plus this.photoBtnClose mais 
Selector(PHO-VIEWER-TOOLBAR-CLOSE) 
```
?

Reply:
`this.PHO_VIEWE_TOOLBAR_CLOSE = '[class*="pho-viewer-toolbar-close"]'` pour le garder dans le constructeur de la page, et `Selector(this.PHO_VIEWE_TOOLBAR_CLOSE)` pour l'appeller, ca marche sans problème.
Si ca parle plus, je peux les changer!

Reply:
c'est purement un avis subjectif. Pas besoin de perdre du temps là-dessus c'était juste pour voir vos avis. cc @y-lohse @cedricmessiant

Reply:
Je pense que plus ca va aller, moins on va se servir des classes comme sélecteurs… donc pas super fan d'abstraire ca. Mais dispo pour en rediscuter plus en détails si besoin.